createtables.sql

```diff
@@ -60,6 +60,17 @@ CREATE TABLE arduino_datalogs (
   current INTEGER
 );
 
+-- -----------------------------------------------------
+-- Table acutators
+-- -----------------------------------------------------
+
```

Comment:
maybe more explicit term for the table.
Also, add in the recent table created in Google drive, in tab New_DB_ASPire :  https://docs.google.com/spreadsheets/u/2/d/1QuP39-nvPkAPv7n1L8Q06pufJ0cedYxDur2Ux6r_IL4/edit?usp=drive_web